test/mitmproxy/contentviews/test_image_parser.py

```diff
@@ -0,0 +1,14 @@
+import io
+
+from mitmproxy.contentviews.image import image_parser
+from mitmproxy.test import tutils
+
+def test_png_parser():
```

Comment:
The `png.ksy` given in the `kaitaistruct` formats doesn't deal with `iTXt` and `zTXt` chunks. I'll try to modify `png.ksy` and compile and use that one.